Editor/Resources/EditorWindow/Pages/AnywherePage.uxml

```diff
@@ -10,45 +10,56 @@
-            <ui:VisualElement name="AnywhereFleetName" class="form-row">
-                <ui:Label name="AnywherePageConnectFleetLabel" class="form-row__label" text="Fleet Name"/>
-                <ui:DropdownField name="AnywherePageConnectFleetDropdown" class="form-row__input dropdown"/>
-            </ui:VisualElement>
-            <ui:VisualElement name="AnywherePageConnectFleetID" class="form-row form-row__hidden">
-                <ui:Label name="AnywherePageConnectFleetIDLabel" class="form-row__label" text="Fleet ID"/>
-                <ui:Label name="AnywherePageConnectFleetIDDisplay" class="form-row__text-display" text="1234567890"/>
-            </ui:VisualElement>
-            <ui:VisualElement name="AnywherePageConnectFleetStatus" class="form-row form-row__hidden">
-                <ui:Label name="AnywherePageConnectFleetStatusLabel" class="form-row__label" text="Fleet status"/>
-                <ui:Label name="AnywherePageConnectFleetStatusDisplay" class="form-row__text-display" text="Connected"/>
-            </ui:VisualElement>
-            <ui:VisualElement name="AnywherePageConnectFleetButtons" class="form-row form-row__hidden">
-                <ui:VisualElement class="form-row__input form-row__input--no-label">
-                    <ui:Button name="AnywherePageConnectFleetNewButton" class="button button--primary" text="Create New Anywhere Fleet"/>
+        <ui:VisualElement name="AnywherePageConnectFleetContainer">
+            <ui:Foldout name="AnywherePageCreateFleetTitle" class="foldout" text="Connect to Anywhere Fleet">
+                <ui:VisualElement name="AnywherePageFleetName" class="form-row">
+                    <ui:Label name="AnywherePageConnectFleetName" class="form-row__label" text="Fleet Name"/>
+                    <ui:VisualElement class="form-row__input">
+                        <ui:TextField name="AnywherePageCreateFleetInput" class="text-input" value="GameName-AnywhereFleet"/>
+                        <ui:Label name="AnywherePageCreateFleetHint" class="input-hint" text="Fleet Name must have 1–1024 characters. Valid characters are A-Z, a-z, 0-9, _ and - (hyphen)"/>
+                    </ui:VisualElement>
                 </ui:VisualElement>
-            </ui:VisualElement>
-        </ui:Foldout>
+                <ui:VisualElement class="form-row">
+                    <ui:VisualElement class="form-row__input form-row__input--no-label">
+                        <ui:Button name="AnywherePageCreateFleetButton" class="button button--primary" text="Create Anywhere Fleet"/>
+                        <ui:Button name="AnywherePageCreateFleetCancelButton" class="button" text="Cancel"/>
+                    </ui:VisualElement>
+                </ui:VisualElement>
+            </ui:Foldout>
+            <ui:Foldout name="AnywherePageConnectFleetTitle" class="foldout" text="Connect to Anywhere Fleet">
+                <ui:VisualElement name="AnywhereFleetName" class="form-row">
```

Comment:
Please rename to `AnywherePageConnectFleetName`. Thanks!

Reply:
This is done in #110